tools/export_slow_tests.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python
+
+import argparse
+import json
+import os
+from tools.stats_utils.s3_stat_parser import get_previous_reports_for_branch, Report, Version2Report
+from typing import cast, Dict, Tuple, List
+
+SLOW_TESTS_FILE = '.pytorch-slow-tests'
+SLOW_TEST_CASE_THRESHOLD = 60.0
+
+
+def get_test_case_times() -> Dict[str, float]:
+    # an entry will be like ("test_doc_examples (__main__.TestTypeHints)" -> (current_avg, # values))
+    reports: List[Report] = get_previous_reports_for_branch('origin/viable/strict', "")
+    test_names_to_times: Dict[str, Tuple[float, int]] = dict()
+    for report in reports:
+        if 'format_version' not in report:  # version 1 implicitly
+            raise RuntimeError("S3 format currently handled is version 2 only")
+        else:
```

Comment:
Please consider getting rid of `else:` and reducing the indent by doing something like:
```
if report.get('format_version', 1) < 2:
   raise "Only V2 is supported"

# Else branch goes in here without indent
```